src/data/markdown/translated-guides/en/01 Getting started/04 Results Output.md

```diff
@@ -168,15 +110,13 @@ The available built-in outputs are:
 
 ### Multiple outputs
 
-You can also send metrics simultaneously to several outputs.
-To do so, use the CLI `--out` flag multiple times.
+You can also send metrics simultaneously to several outputs
```

Comment:
Another suggestion,  removing here the H3 heading and code example. 

I could be as a note, below the list of outputs.

> You can also send metrics simultaneously to several outputs by using multiple CLI --out flags.

Reply:
Good idea, fixed.